src/todt.d

```diff
@@ -1505,7 +1505,7 @@ extern (C++) class TypeInfoDtVisitor : Visitor
             Parameter arg = (*tu.arguments)[i];
 
             genTypeInfo(arg.type, null);
-            dtxoffDsymbol(dtb, arg.type.vtinfo, 0, dataSymbolRefs);
+            dtxoffDsymbol(dtbargs, arg.type.vtinfo, 0, dataSymbolRefs);
```

Comment:
Good find! I looked at the same line but it seems I'm to tired to see the mistake.